hack/quick-build-gke.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
It's a good idea - the EKS one might be very similar (or the same) because we are just pushing to an actual registry. I can add one/rename this one to be generic the next time we do something on EKS.